UltraStar Play/Assets/Common/UI/StyleSheetControl.cs

```diff
@@ -47,37 +45,88 @@ void Start()
         AddScreenSpecificStyleSheets();
         UpdateThemeSpecificStyleSheets();
 
-        settings.GraphicSettings.ObserveEveryValueChanged(graphicSettings => graphicSettings.themeName)
+        settings.GraphicSettings.ObserveEveryValueChanged(graphicSettings => graphicSettings.CurrentThemeName)
             .Subscribe(_ => UpdateThemeSpecificStyleSheets())
             .AddTo(gameObject);
     }
 
-    private void UpdateThemeSpecificStyleSheets()
+    /// <summary>Queries all the relevant UI controls and change their style according to the currently loaded theme.</summary>
```

Comment:
- IMO the XML doc style of C# is unnecessarily verbose. Please prefer JavaDoc / JSDoc style.
- My IDE (Rider) can handle both, JavaDoc and the XML doc style. Your IDE can probably also handle both styles.